src/mono/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -267,7 +267,6 @@
       <Compile Include="$(BclSourcesRoot)\System\Threading\Monitor.Mono.cs" />
       <Compile Include="$(BclSourcesRoot)\System\Threading\ObjectHeader.Mono.cs" />
       <Compile Include="$(BclSourcesRoot)\System\Threading\Thread.Mono.cs" />
-      <Compile Include="$(BclSourcesRoot)\System\Threading\ThreadPool.Mono.cs" />
```

Comment:
I don't think this is right, sorry I had misworded my comment and edited it yesterday. I meant to delete the method, not the file.

Reply:
Ah yea but without the method the file was only containing the class declaration, wouldn't it be better to delete it in that case?

Reply:
Ah, that's what I forgot about. Maybe I did mean to delete the file. Sorry for the confusion.